### PR TITLE
Small navigation improvements

### LINKED ISSUE #198: Small navigation improvements
Follow-up to #184

- [ ] let the "Join the Conversation" link sit at the bottom for non-admins, and push it up with the padding you have here for logged-in admins
- [ ] align the left side of the word "Conversation" with "Join" in the line above

_Originally posted by @ckadner in https://github.com/machine-learning-exchange/mlx/issues/184#issuecomment-910549650_

@drewbutlerbb4

### PR DESCRIPTION
**Pull request by Prasantacharya**
Resolves #198 
For non-logged in users, the "join the conversation" button sits at the bottom, but for admins, it is in its usual spot
Also the text now aligns with the paragraph instead of the icon

### CODE REVIEW

**Comment by Prasantacharya** [comment]
@ckadner ive shifted down the icons a bit. Altho when i built the site to test, I didnt have the white pullet points, which I assume is what I would be centering with?

**Comment by ckadner** [comment]
The speech bubble icon is aligned nicely now, but now the bullet point is sitting too low. 

Screen Shot 2021-10-19 at 4 32 32 PM

**Comment by ckadner** [comment]
@Prasantacharya -- could you sign your commits to get the DCO check to pass?

**Comment by ckadner** [comment]
@Prasantacharya -- did you ever get a chance to find out why the bullet point shows up out of line? I [screenshoted](https://github.com/machine-learning-exchange/mlx/pull/238#issuecomment-947185965) that on Firefox

**Review by ckadner — CHANGES_REQUESTED** [review]
@Prasantacharya -- if you can find out how to vertically align (center) the icon and text with the bullet point that would be great.

**Comment by Prasantacharya** [comment]
image
this is the closest ive gotten it to, does this look right?

**Comment by ckadner** [comment]
> image this is the closest ive gotten it to, does this look right?

Yeah, did this require further code changes? Did you push those to this PR?

File: README.md [hunk 1]
```diff
@@ -13,6 +13,9 @@ Allows upload, registration, execution, and deployment of:
  - Datasets
  - Notebooks
 
+For more details about the project please follow this [announcement blog post](https://lfaidata.foundation/blog/2021/09/28/machine-learning-exchange-mlx/).
```
Comment (ckadner) [inline README.md:16]: This change here seems unrelated. Could you rebase your PR or merge in the latest changes from `main`?

**Review by ckadner — COMMENTED** [review]
/lgtm

Thanks @Prasantacharya

**Pull request merged**